### PR TITLE
Correcting typos for the French version of the tutorial

### PR DESCRIPTION
**Pull request by Wagyx**
Hi DjangoGirls team,

I went through the French tutorial this week and could not help but notice some mistakes in the text.
They are mainly typos, redundant or missing words. I have corrected everything I could see.

By the way, this tutorial is really good, I would like to thank you all for your hard work.

### CODE REVIEW

**Comment by olasitarska** [comment]
@patjouk Could you review whenever you've got time? :)

File: fr/css/README.md [hunk 1]
```diff
@@ -144,15 +144,15 @@ Bravo ! Peut-être que nous pourrions un peu aérer notre page web en augmentant
...
+```
 
 Ajoutez ceci à votre fichier CSS, sauvegardez-le et voyons le résultat !
 
 ![Figure 14.3][6]
 
  [6]: images/margin2.png
 
-Et si nous changions aussi la police de caractères de notre entête ? Collez ceci dans la partie `<head>` de votre fichier `blog/templates/blog/post_list.html` :
+Et si nous changions aussi la police de caractères de notre en-tête ? Collez ceci dans la partie `<head>` de votre fichier `blog/templates/blog/post_list.html` :
```
Comment (patjouk) [inline fr/css/README.md]: We will keep "entête". Can you change it back?

**Comment by patjouk** [comment]
Hi. We discussed the use of reforme 1990 spelling versus old spelling and decided to go for reforme. I'll check the tutorial again to move it back to reforme spelling and put a note about this in the tutorial introduction.
To be able to merge your PR, I'll need you to remove changes linked to old spelling (apparaître/apparaitre, en-tête/entête, etc).
Thanks! :sparkles:

**Comment by Wagyx** [comment]
Sure, I will do it as soon as I can.
Thanks for the review.

**Comment by Wagyx** [comment]
It's done. I may have even corrected some extra "apparaître", "rafraîchir", "connaître", "entraîner".
I have omitted all the "chaîne" though.

**Comment by patjouk** [comment]
Awesome, thanks <3
I'll take care of chaine :wink: 
Can you fix the merge conflict? I'll merge your PR as soon as it's done. Thanks!

**Comment by Wagyx** [comment]
It's done.
It took me some time to figure out what to do since it was a first for me.
I learned a lot. Thank you.

**Comment by patjouk** [comment]
Amazing job! Thanks a lot :sparkles:

**Pull request merged**